DESCRIPTION

```diff
@@ -35,7 +35,8 @@ Imports:
     cli,
     countrycode,
     dplyr,
-    stringi
+    stringi,
+    tibble
```

Comment:
`tribble()` is also re-exported y `dplyr`, so one can simplify things by using `dplyr::tribble()`

Reply:
noted, will do